src/services/axios.ts

```diff
@@ -1,11 +1,12 @@
 import axios from 'axios';
-
+import { store } from '../App'
+import { notLoggedInException } from '../store/user/actions';
 
 const axiosInstance = axios.create();
 
 axiosInstance.interceptors.response.use((res: any) => {
-  if(res.error.message === 'must be logged in') {
-    
+  if(res.error?.message === 'must be logged in') {
```

Comment:
are we already using `?` (optional chaining) in this project? It is not fully specified and I don't want to hurry to add stuff that are nice but might cause problems down the road, for a not-very-steep-return-of-investment. 
might not be a problem, but would be nice to check if it works on browsers (or just postpone the decision and just go with this for now:
```
if(res && res.error && res.error.message === 'must be logged in') {
```

Reply:
![image](https://user-images.githubusercontent.com/11048153/77149745-f66bf700-6a9a-11ea-90b2-5c9165e04229.png)

Reply:
ok read more, it might be that typescript is compiling this to code that does not use the `?` operator in the browser, so it probably is safe (still need to test, still not sure if testing is worth in this case 😸 )
```
a?.b
```
will be compiled to:
```
a ? a.b : null
```

Reply:
I was pretty amazed that typescript give this power combined with create-react-app, I am using it on my native project and i did double check that it works on create react app as well (even ran a build and inspected the compiled code).
As you mentioned it is compiled for a readable code for cross browsers.